src/main/java/nextstep/courses/domain/ImageSize.java

```diff
@@ -0,0 +1,37 @@
+package nextstep.courses.domain;
+
+import java.util.Objects;
+
+public class ImageSize {
+    private static final long MAX_SIZE_IN_KB = 1024;
+
+    private long size;
```

Comment:
값 객체라면 불변 형식으로 만들어볼까요?

```suggestion
    private final long size;
```